EventEnabledInsurance/prereqs.sh

```diff
@@ -132,6 +132,35 @@ fi
 
 echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
+REPLICATION_USER=$(echo ${namespace}_sor_reeplication_${SUFFIX} | sed 's/-/_/g')
+REPLICATION_PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 32 ; echo)
+REPLICATION_PASSWORD_ENCODED=$(echo -n ${REPLICATION_PASSWORD} | base64)
+
+DB_POD=$(oc get pod -n ${POSTGRES_NAMESPACE} -l name=postgresql -o jsonpath='{.items[].metadata.name}')
```

Comment:
The DB_POD is on line 77 as well. Is this a different pod to that one or can it be removed?

Reply:
Good point, nothing has changed to restart the DB_POD. Those lines have been removed